src/main/kotlin/com/pubnub/api/subscribe/eventengine/configuration/EventEnginesConfImpl.kt

```diff
@@ -0,0 +1,27 @@
+package com.pubnub.api.subscribe.eventengine.configuration
+
+import com.pubnub.api.eventengine.EventEnginesConf
+import com.pubnub.api.eventengine.QueueSinkSource
+import com.pubnub.api.eventengine.Sink
+import com.pubnub.api.eventengine.Source
+import com.pubnub.api.presence.eventengine.effect.PresenceEffectInvocation
+import com.pubnub.api.presence.eventengine.event.PresenceEvent
+import com.pubnub.api.subscribe.eventengine.effect.SubscribeEffectInvocation
+import com.pubnub.api.subscribe.eventengine.event.SubscribeEvent
+
+class EventEnginesConfImpl : EventEnginesConf {
```

Comment:
I have proposal how we can do keep the "configurations" for both presence and subscribe more separate. I will have it in one of the commits